main.tf

```diff
@@ -232,6 +232,13 @@ resource "aws_lb_listener" "lb_http_listeners" {
       type             = "forward"
     }
   }
+  
+  dynamic "lifecycle" {
+    for_each = (lookup(each.value, "type", "") == "" || lookup(each.value, "type", "") == "forward") ? [1] : []
+    replace_triggered_by {
+      target_group_id = aws_lb_target_group.lb_http_tgs[each.key].id
+    }
+  }
```

Comment:
Hi, shouldn't this apply for the https listener too?

Reply:
Updated